Cargo.toml

```diff
@@ -28,3 +28,6 @@ serde_json = { version = "1.0", features = ["raw_value"] }
 
 [dev-dependencies]
 ctor = "0.2.5"
+assert_cmd = "2.0.12"
+predicates = "3.0.4"
+escargot = "0.5.8"
```

Comment:
As we discussed offline, if it's easy to work around, let's try to remove crates or try to make sure they are `no_std` compatible. If there is no good alternative, it's fine to add them for now.

Reply:
assert_cmd [doesn't support running with `features`](https://github.com/assert-rs/assert_cmd/issues/59#issuecomment-472684304), using escargot seems mandatory, nevertheless, it is `no std` compatible.